setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub==1.25.2',
+        'PyGithub==1.25.2-squash',
```

Comment:
i tried this dependency_links option but couldnt get it working. 
currently that squash-version i am manually installing right now.
I added to the merge-pr job a line right before it does the pip install of github-pr that performs a manual install from my fork.
Tried to get a screenshot here, couldn't